tests/unit/test_peak_fit_engine.py

```diff
@@ -337,6 +344,13 @@ def test_2_gaussian_1_subrun():
     assert eff_param_values[0, 0] == param_values[1, 0]   # center
     assert abs(eff_param_values[0, 0] - peak_centers[0]) < 2e-2, 'Peak center is not correct'
 
+    # Parameters verified
```

Comment:
Rather than hand-copying the values, either parameterize the function parameters so they come from the test set-up, or make them file constants than can be re-used here